setup.py

```diff
@@ -29,7 +29,8 @@
 echionmodule = Extension(
     "echion.core",
     sources=["echion/coremodule.cc"],
-    include_dirs=["."],
+    include_dirs=[".", "vendor/libunwind-1.6.2/include"],
+    library_dirs=["vendor/libunwind-1.6.2/src/.libs"],
```

Comment:
Do we need some CI changes for this to work in this repo?

Reply:
I forget the exact problem.  I don't remember whether it was binary size or just the availability of the library in CI.  I'd  suggest adding it back in and taking it from there.